File: Server/Cinema/CinemaApp.Application/DTOs/Search/MultipleSearchResult.cs
using CinemaApp.Application.DTOs.Cinema;
using CinemaApp.Application.DTOs.City;
using CinemaApp.Application.DTOs.Movie;

namespace CinemaApp.Application.DTOs.Search
{
    public class MultipleSearchResult
    {
        public IEnumerable<CityDto> Cities { get; set; }
        public IEnumerable<SearchCinemaDto> Cinemas { get; set; }
        public IEnumerable<SearchMovieDto> Movies { get; set; }
    }
}

File: Client/cinema-app/src/app/core/models/search/multiple-search-result.ts
import { SearchCinema } from '@core/models/cinema/search-cinema';
import { City } from '@core/models/city/city';
import { SearchMovie } from '@core/models/movie/search-movie';

export interface MultipleSearchResult {
  movies: SearchMovie[],
  cinemas: SearchCinema[],
  cities: City[]
}
